import_busstops.py

```diff
@@ -5,6 +5,11 @@
 
 class BusImportCommand(CSVImportCommand):
 
+    def __init__(self, *kargs, **kwargs):
+        self.dry_run = kwargs.pop('dry_run')
+        super(BusImportCommand, self).__init__(
+            *kargs, **kwargs)
+
     def process_row(self, row):
         try:
```

Comment:
should we remove this try/except then? (since the exception wasn't even raised here)